Polish release notes 1.x (#8358)
dotnet/core: release-notes/2.2/2.2-known-issues.md

**Note this has been resolved as of January 2019.**

> An update to the ASP.NET Core Module (ANCM) has released for .NET Core 2.2, but is not yet available. This update adds support for hosting applications in the IIS worker process. The updated ANCM is expected to be deployed to Azure App Services later in December 2018. In the meantime, the following workaround, applied to the projects Memory<byte> file, can be applied which will cause ASP .NET Core 2.2 projects to use the existing ASP.NET Core Module.

Reviewer: ```suggestion
An update to the ASP.NET Core Module (ANCM) has released for .NET Core 2.2, but is not yet available. This update adds support for hosting applications in the IIS worker process. The updated ANCM is expected to be deployed to Azure App Services later in December 2018. In the meantime, the following workaround, applied to the projects csproj file, can be applied which will cause ASP .NET Core 2.2 projects to use the existing ASP.NET Core Module.
```

Change this section ...

> ```C#

Reviewer: ```suggestion
```xml
```

to the following ...

> ```C#

Reviewer: ```suggestion
```xml
```
